.env-example

```diff
@@ -0,0 +1,52 @@
+# domain name
+# set to localhost if developing locally
+DOMAIN=perf.test.fedcloud.edu
+
+# NGINX update credentials
+NGINX_API_USER=user
+NGINX_API_PASS=pass
+
+# letsencrypt cert email
+# you may leave this blank, but it is strongly recommended to add one
+EMAIL=
+
+# certbot staging mode
+# Set to 1 if you're testing your setup to avoid hitting request limits
+STAGING=0
+
+# encryption key for flask session cookie encryption
+# if the file is blank, a random key will be generated every instance and will not persist
+# (leave this to default for docker)
+EOSC_PERF_COOKIE_CRYPT_KEY_PATH=/run/secrets/cookie_secret
+
+# EGI-AAI OIDC configuration
+#  Client secret to use (leave this to default for docker)
+EOSC_PERF_OIDC_CLIENT_SECRET_PATH=/run/secrets/oidc_client_secret
```

Comment:
If is is going to be static or dependent on the docker-compose config, I would move it to the docker-compose itsef

Reply:
I was considering the scenario where you'd want to run it outside docker and didn't docker(-compose) setting up the files there for you, I can let it be hardcoded though